cfme/tests/configure/test_log_depot_operation.py

```diff
@@ -142,9 +143,22 @@ def get_ftp():
                 ftp.recursively_delete()    # And so on ...
 
             """
-            return FTPClient(ftp_credentials["hostname"],
-                             ftp_credentials["username"],
-                             ftp_credentials["password"])
+            # Condition to check if it anonymous ftp or not, use default anonymous user
+            if depot_type == "anon_ftp":
+                ftp_user_name = "anonymous"
+                ftp_password = ""
+                # in case if it's anonymous connection cfme works only with hardcoded "incoming" directory
+                upload_dir ="incoming"
```

Comment:
We want to have a conditional using the `utils.blockers.BZ` class here that would stop using the `'incoming'` folder once the issue is fixed. I presume that the fix in manageiq for the BZ raised will be to remove this hard-coded requirement for an 'incoming' folder.